site/packages/aa-accounts/light-account/index.md

```diff
@@ -56,10 +57,18 @@ const ownerAddress = await smartAccountClient.account.getOwnerAddress();
 
 // transfer ownership
 const newOwner = LocalAccountSigner.mnemonicToAccountSigner(NEW_OWNER_MNEMONIC);
-const hash = smartAccountClient.transferOwnership({
+const hash = await smartAccountClient.transferOwnership({
   newOwner,
   waitForTxn: true, // wait for txn with UO to be mined
 });
+
+// after transaction is mined on the network,
+// create a new light account client for the transferred Light Account
+const transferredClient = await createLightAccountClient({
+  transport: custom(smartAccountClient),
+  chain: smartAccountClient.chain,
+  signer: newOwner,
```

Comment:
You also need to pass in the old account address as an override here

Reply:
ah forgot